h/accounts/models.py

```diff
@@ -106,6 +124,12 @@ def subscriptions(self, value):
             self.status = (self.status or 0) & ~0b100
 
 
+def _username_to_uid(username):
+    # We normalise usernames by dots and case in order to discourage attempts
+    # at impersonation.
+    return username.replace('.', '').lower()
```

Comment:
I still think we should include underscores here too, if we're going this direction. Seems a bit half baked to normalise one non alpha-numeric but not the other.